src/odemis/driver/test/piezomotor_test.py

```diff
@@ -35,9 +35,11 @@
 # Export TEST_NOHW=1 to force using only the simulator and skipping test cases
 # needing real hardware
 TEST_NOHW = (os.environ.get("TEST_NOHW", 0) != 0)  # Default to Hw testing
+MULTIPLE_AXES = (os.environ.get("MULTIPLE_AXES", 0) != 0)  # if available, test 2 axes
 
 TEST_SPEED = 0.001  # m / s
 AXIS_NUM = 1
+AXIS_NUM2 = 2
```

Comment:
Is that really necessary? In practice, for a lot of drivers, when testing with a specific hardware you have  to adjust the test case configuration. I'd suggest to do the same as in most other drivers: have a KWARGS variable, instead of TEST_SPEED, AXIS_NUM and AXIS_NUM2. And have it configured for 2 axes. If at some point we need to test on a hardware with a single axis, we'll disable the second axis config.